foundry.toml

```diff
@@ -1,4 +1,5 @@
 [profile.default]
+libs = ["lib"]
```

Comment:
This change is to make the versioning independent from usage of the repository. For example, try (without this line) to do:
```bash
rm -rf lib
forge install
git status
```